docker/.env

```diff
@@ -406,5 +406,8 @@ MF_GRAFANA_PORT=3000
 MF_GRAFANA_ADMIN_USER=mainflux
 MF_GRAFANA_ADMIN_PASSWORD=mainflux
 
+###
+MF_CLI_LOG_LEVEL=debug
```

Comment:
@WashingtonKK  
 Instead of having Env Variable for LOG LEVEL , Passing command line arguments 
 like `mainflux-cli --debug`  will be more useful